workorders.lic

```diff
@@ -836,9 +836,9 @@ class WorkOrders
       when 'You realize you have items bundled with the logbook'
         DRC.bput('untie my logbook', 'You untie')
         if DRC.left_hand.include?('logbook')
-          fput("drop my #{DRC.right_hand}")
+          fput("drop my #{DRC.right_hand}") unless DRCI.dispose_trash(DRC.right_hand)
         else
-          fput("drop my #{DRC.left_hand}")
+          fput("drop my #{DRC.left_hand}") unless DRCI.dispose_trash(DRC.left_hand)
```

Comment:
I believe `DRCI.dispose_trash` already drops an item if no trashcan is available.

https://github.com/rpherbig/dr-scripts/blob/144fd57703fb334ecb724358f31c9453ff9b6155/common-items.lic#L413

Reply:
> I believe `DRCI.dispose_trash` already drops an item if no trashcan is available.
> 
> https://github.com/rpherbig/dr-scripts/blob/144fd57703fb334ecb724358f31c9453ff9b6155/common-items.lic#L413

Good suggestion. Thank you!